src/main/kotlin/com/dansplugins/factionsystem/law/JooqMfLawRepository.kt

```diff
@@ -38,15 +48,56 @@ class JooqMfLawRepository(
     }
 
     override fun delete(id: MfLawId) {
+        delete(getLaw(id)!!)
+    }
+
+    override fun delete(law: MfLaw) {
         dsl.deleteFrom(MF_LAW)
-            .where(MF_LAW.ID.eq(id.value))
+            .where(MF_LAW.ID.eq(law.id.value))
+            .execute()
+        dsl.update(MF_LAW)
```

Comment:
Could this be achieved without updating all the numbers each time? Perhaps use it as a sort index for display and then treat the display number differently?

Reply:
I don't see how I would do that, but you're welcome to make a commit to my fork if you know how to.